### PR TITLE
Create landing page, reorganize website, and add funding acknowledgment

### LINKED ISSUE #28: Announce NSF CSSI grant and update funding acknowledgements
The NSF CSSI grant is important news, so it would be well worth adding an announcement on our front page (as well as sending out an email to the list).  

While doing this, we should update the funding acknowledgements to reflect the NSF CSSI grant.  This would be essentially the same as https://github.com/PlasmaPy/PlasmaPy/issues/679, but for the website.  For the text, we can cut and paste the updated acknowledgements from https://github.com/PlasmaPy/PlasmaPy/pull/688.  It'd probably be good to create a funding announcements page.  

I'll assign this to myself, and hopefully we can get this merged before the APS DPP meeting begins on October 21.  This is probably of interest to @tulasinandan and @rocco8773 too.

### LINKED ISSUE #30: Create a landing page for website
Right now the front page of our website has announcements for different releases.  An alternative would be to have a landing page (like the [landing page for Astropy](https://www.astropy.org/)) that describes the project and points users on where to go, rather than having a series of announcements.  (An advantage of that would be that we wouldn't have to create a new announcement for every release!)  While we're doing this, we should also update our funding acknowledgements as per #28.  

Thanks to @rocco8773 (I think) for bringing this up!

### PR DESCRIPTION
**Pull request by namurphy**
The purpose of this pull request is to make a landing page for the website, akin to [Astropy's landing page](https://www.astropy.org/).  Thus far I moved the blog posts to `news/`, changed the order of the tabs, and created `web/pages/index.md` for the new landing page.  We still need to discuss what would go in there.  

Closes #30.  Closes #28.

### CODE REVIEW

File: web/conf.py [hunk 1]
```diff
@@ -145,34 +145,31 @@
...
                 (FEEDBACK_BOX, "Suggestion box"),
                 (TWITTER, "Twitter"),
-            ), "Contact"),
-        ((
-            (TELECON_LINK, "Teleconference room (Jitsi Meet)"),
-            (TELECON_CALENDAR, "Teleconference calendar & dates"),
-            (TELECON_NOTES, "Notes from weekly meetups"),
-        ), "Teleconferences"),
+#                (TELECON_LINK, "Teleconference room (Jitsi Meet)"),
+                (TELECON_CALENDAR, "Calendar"),
```
Comment (StanczakDominik) [inline web/conf.py]:
```suggestion
                (TELECON_CALENDAR, "Calendar"),
                ("https://plasmapy.discourse.group/", "Discussion forum"),
```

**Comment by namurphy** [comment]
Oh, good catch!  I had forgotten about the Discourse group.

**Comment by namurphy** [comment]
This looks pretty close to ready to merge so that @rocco8773 can finish working on a meeting page.  We will probably want to make more updates to the content in the near future, but this looks like a good spot to merge it for now.

**Comment by namurphy** [comment]
Merging this now, but we will likely be continuing to make changes in the next few weeks.

**Pull request merged**